File: app/src/main/java/com/sctdroid/autosigner/fragments/StatusFragment.java
package com.sctdroid.autosigner.fragments;

import android.app.Fragment;
import android.os.Bundle;
import androidx.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.sctdroid.autosigner.R;
import com.sctdroid.autosigner.presentation.ui.views.PaginationListview;

/**
 * Created by lixindong on 6/28/16.
 */
public class StatusFragment extends Fragment {
    View mContentView = null;
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        if (mContentView == null) {
            mContentView = inflater.inflate(R.layout.fragment_status, container, false);
        }
        return mContentView;
    }

    PaginationListview mListview;
    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        mListview = (PaginationListview) mContentView.findViewById(R.id.listview);
//        mListview.setInteractor(new PaginationInteractorImpl(null,
//                new FollowerPaginationRepository(getActivity(), Constants.APP_KEY, AccessTokenKeeper.readAccessToken(getActivity()))));
//        mListview.showLoadMore();
//        mListview.setAdapter(new PaginationAdapter(getActivity(), FriendItemView.class));
    }
}

File: app/src/main/java/com/sctdroid/autosigner/fragments/StatusFragment.kt
package com.sctdroid.autosigner.fragments

import android.os.Bundle
import android.os.Parcelable
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.sctdroid.autosigner.R
import com.sina.weibo.sdk.openapi.models.Status
import kotlinx.android.synthetic.main.fragment_status.*
import org.parceler.Parcels

class StatusFragment : BaseNavigationFragment() {
    companion object {
        const val KEY_STATUS = "key_status"
        fun newInstance(status: Status): StatusFragment {
            val pStatus = Parcels.wrap(status)
            return StatusFragment().apply {
                arguments = Bundle().apply {
                    putParcelable(KEY_STATUS, pStatus)
                }
            }
        }
    }

    private lateinit var status: Status
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val pStatus = arguments?.getParcelable<Parcelable>(KEY_STATUS)
        status = Parcels.unwrap<Status>(pStatus)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
        return inflater.inflate(R.layout.fragment_status, container, false)
    }

    override fun onViewCreated(view: View) {
        statusItem.bind(status)
    }
}